src/explore-education-statistics-common/src/services/types/releaseType.ts

```diff
@@ -7,4 +13,12 @@ export const releaseTypes = {
   ManagementInformation: 'Management information',
 } as const;
 
+export const releaseTypeComponents = {
+  NationalStatistics: NationalStatisticsSection,
+  OfficialStatistics: OfficialStatisticsSection,
+  ExperimentalStatistics: ExperimentalStatisticsSection,
+  AdHocStatistics: AdHocStatisticsSection,
+  ManagementInformation: ManagementInformationSection,
+};
```

Comment:
Doesn't seem appropriate to put this in here. This is the `services` directory, so we shouldn't need to put React components in here. 

These are also quite specific to the section components for the release pages, which doesn't make a lot of sense for a file that will be used more generally.

Would suggest we just create a dedicated `ReleaseTypeSection` component that takes a `type` prop and wraps these other sections. We might even want to just move all the other section components into the same file for simplicity (up to you).